marimo/_plugins/stateless/audio.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2023 Marimo. All rights reserved.
+from __future__ import annotations
+
+import base64
+import io
+from typing import Union
+
+from marimo._output.builder import h
+from marimo._output.hypertext import Html
+from marimo._output.rich_help import mddoc
+
+
+@mddoc
+def audio(
+    src: Union[str, io.IOBase],
+) -> Html:
+    """Render an audio file as HTML.
+
+    **Example.**
+
+    ```python3
+    mo.audio(
+        src="/recording.wav"
```

Comment:
Let's use a URL. Otherwise some users might think you can pass a path to a file on your local filesystem.

Maybe

```python3
mo.audio(
    src="https://upload.wikimedia.org/wikipedia/commons/8/8c/Ivan_Ili%C4%87-Chopin_-_Prelude_no._1_in_C_major.ogg"
)
```